--- a/crates/cheatnet/src/runtime_extensions/outer_call_runtime_extension/execution/entry_point.rs
+++ b/crates/cheatnet/src/runtime_extensions/outer_call_runtime_extension/execution/entry_point.rs
@@ -198,6 +198,6 @@
     context.revert_infos.0.push(EntryPointRevertInfo::new(
         entry_point.storage_address,
-        class_hash,
+        storage_class_hash,
         context.n_emitted_events,
         context.n_sent_messages_to_l1,
     ));
